sdk/file_sdk/OPCFileFormat/OPCFileFormat.pro

```diff
@@ -5,11 +5,20 @@ TARGET    = modopcfile
 TEMPLATE = lib
 CONFIG += staticlib warn_on c++11 debug_and_release
 INCLUDEPATH += $$REPO_ROOT/sdk/rmscrypto_sdk/CryptoAPI $$REPO_ROOT/sdk/file_sdk
+INCLUDEPATH += $$REPO_ROOT/sdk/file_sdk $$REPO_ROOT/sdk/file_sdk/Common
+
+win32:INCLUDEPATH += $$REPO_ROOT/third_party/include/Libgsf
+
+win32:LIBS += -L$$REPO_ROOT/third_party/lib/Libgsf -llibgsf-1-114 -lzlib1\
+               -llibgobject-2.0-0\
+               -llibglib-2.0-0 -llibbz2-1
```

Comment:
what about linux?

Reply:
added one for linux, lets see if this will fix issues

---
In reply to: [139360940](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139360940) [](ancestors = 139360940)